src/Renderer/AbstractRenderer.php

```diff
@@ -323,18 +359,20 @@ public function getAutomaticRenderError()
 
     /**
      * Set the barcode object
-     * @param  Object\ObjectInterface $barcode
+     *
+     * @param ObjectInterface $barcode
      * @return self Provides a fluent interface
      */
-    public function setBarcode(ObjectInterface $barcode)
```

Comment:
This parameter type should be restored, since `AbstractRenderer` can be extended, and the removal will break existing implementations